Sources/Moya/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -87,21 +90,32 @@ private extension NetworkLoggerPlugin {
         return output
     }
 
-    func logNetworkResponse(_ response: HTTPURLResponse?, data: Data?, target: TargetType) -> [String] {
-        guard let response = response else {
-           return [format(loggerId, date: date, identifier: "Response", message: "Received empty network response for \(target).")]
+    func logNetworkResponse(_ response: Response, target: TargetType, isFromError: Bool) -> [String] {
+        guard let httpResponse = response.response else {
+            return [format(loggerId, date: date, identifier: "Response", message: "Received empty network response for \(target).")]
         }
 
         var output = [String]()
 
-        output += [format(loggerId, date: date, identifier: "Response", message: response.description)]
+        output += [format(loggerId, date: date, identifier: "Response", message: httpResponse.description)]
 
-        if let data = data, let stringData = String(data: responseDataFormatter?(data) ?? data, encoding: String.Encoding.utf8), isVerbose {
-            output += [stringData]
+        if let stringData = String(data: responseDataFormatter?(response.data) ?? response.data, encoding: String.Encoding.utf8),
+            isFromError || isVerbose {
```

Comment:
hmm, why does it matter if the response is from error or not?

Reply:
I guess it's based on the way I work which, as you ask the question, may not be the same as most of other users :p
I don't want the responses's body to be logged every time because it may contains a lot of informations that I don't care about if the request suceeded, so most of the time the verbose mode is off.
But what I do care about is to know why a certain request failed, and useful informations are often sent by APIs in the body, which is often a simple object with some properties: not such risk at clogging the console here.

Reply:
Ah, now it makes sense! Thanks for the explanation.

Reply:
I agree and will remove the forced log in error. About the log level, yes i've thought about that too, but it may deserve its own PR. I'll work on it.